src/main/kotlin/com/jetbrains/snakecharm/lang/parser/SnakemakeExpressionParsing.kt

```diff
@@ -390,6 +394,9 @@ class SnakemakeExpressionParsing(context: SnakemakeParserContext) : ExpressionPa
         } else if (!atAnyOfTokensSafe(PyTokenTypes.PLUS)) {
             statementBreakMarker?.rollbackTo()
         } else {
+            if (incorrectUnindentMarker != null) {
+                incorrectUnindentMarker?.error(SnakemakeBundle.message("PARSE.incorrect.unindent"))
+            }
```

Comment:
Please simplify this `if` in 397...399. Here you do double-check for nullable value.